lib/lightning_web/live/components/deletion_modal.ex

```diff
@@ -181,4 +233,68 @@ defmodule LightningWeb.Components.UserDeletionModal do
     </div>
     """
   end
+
+  def render(%{project: %Project{}} = assigns) do
+    ~H"""
+    <div id={"project-#{@id}"}>
+      <PetalComponents.Modal.modal
+        max_width="sm"
+        title="Delete project"
+        close_modal_target={@myself}
+      >
+        <.form
+          :let={f}
+          for={@scheduled_deletion_changeset}
+          phx-change="validate"
+          phx-submit="delete"
+          phx-target={@myself}
+          id="scheduled_deletion_form"
+        >
+          <span>
+            This project and all data associated to it (workflows, jobs, project users, project credentials, ...) will be deleted.
+          </span>
```

Comment:
"This project and all data associated to it (workflows, jobs, project users, project credentials, ...) will be deleted. Please make sure none of the workflows in it are still in use."

Reply:
Done